redistrib/command.py

```diff
@@ -80,11 +80,11 @@ def addslots(t, begin, end):
     addslots(t, begin, end)
 
 
-def create(host_port_list, max_slots=1024):
+def create(host_port_list, password=None, max_slots=1024):
```

Comment:
The new optional parameter should be append after existing parameters for API compatible.